plugins/aws/aws.plugin.zsh

```diff
@@ -198,13 +198,32 @@ compctl -K _aws_profiles asp acp aws_change_access_key
 function aws_prompt_info() {
   local _aws_to_show
   local region="${AWS_REGION:-${AWS_DEFAULT_REGION:-$AWS_PROFILE_REGION}}"
+
   if [[ -n $AWS_PROFILE ]];then
-    _aws_to_show+="${ZSH_THEME_AWS_PROFILE_PREFIX:=<aws:}${AWS_PROFILE}${ZSH_THEME_AWS_PROFILE_SUFFIX:=>}"
+    if [[ -z ${ZSH_THEME_AWS_PROFILE_PREFIX+x} ]]; then # empty string is evaluated as false (ref : https://stackoverflow.com/a/13864829/8556340)
+      ZSH_THEME_AWS_PROFILE_PREFIX="<aws:"
+    fi
+    if [[ -z ${ZSH_THEME_AWS_PROFILE_SUFFIX+x} ]]; then
+      ZSH_THEME_AWS_PROFILE_SUFFIX=">"
+    fi
+    _aws_to_show+="${ZSH_THEME_AWS_PROFILE_PREFIX}${AWS_PROFILE}${ZSH_THEME_AWS_PROFILE_SUFFIX}"
+  fi
+
+  if [[ -z ${ZSH_THEME_AWS_DIVIDER+x} ]]; then
+    ZSH_THEME_AWS_DIVIDER=" "
   fi
+  _aws_to_show+="${ZSH_THEME_AWS_DIVIDER}"
+
   if [[ -n $AWS_REGION ]]; then
-    [[ -n $AWS_PROFILE ]] && _aws_to_show+=" "
```

Comment:
Let's also have this check. It'll prevent adding the divider if the region or profiles are not set.

Reply:
My basic idea was that most users would have both profile and region env. But it seems not great guess so I will change it like this: 

```
_aws_to_show+="${ZSH_THEME_AWS_DIVIDER}" # moves this in the previous commit to the...

  if [[ -n $AWS_REGION ]]; then
    [[ -n $AWS_PROFILE ]] && _aws_to_show+="${ZSH_THEME_AWS_DIVIDER=' ' }" # here
```

What do you think? Let me just push a commit first so you can see it easily...